server/README.md

```diff
@@ -4,75 +4,45 @@
 
-## Packages
+## Dev Setup
+In the website directory copy the .env.txt to the same directory, rename it .env, and then add your configuration. Only STEAM_WEB_API_KEY needs to be updated for development. Open config.js in ./server/config to view default values for each environment.
 
-In the command line, navigate to this directory and run
+From the website directory, run:
 ```
-yarn install
-``` 
-to install all project packages.
-
-## Running a Development Server
-
-### Set up the Database
-For your database, run the following:
-
-```
-CREATE DATABASE momentum;
-CREATE USER mom@localhost;
-GRANT ALL PRIVILEGES ON momentum.* TO mom@localhost;
+docker compose up -d
 ```
-
-This will set up a simple test user for the database. You only need to do this once.
-
-Next, the initial database content will need to be created (tables, constraints, etc.). Run the force sync DB script in the scripts folder to do so:
-
+The initial database content will need to be created (tables, constraints, etc.). Run the force sync DB script to do so once the database container is initialized:
 ```
-node <path-to-file>/force_sync_db.js
+docker exec website-api-1 node ../scripts/force_sync_db.js
 ```
-
-#### Applying Database Updates
-
-If updates or additions are made to the database design after you create the database tables for the first time, you will most likely have to apply these updates to your local databases. To do this, simply run the force_sync_db.js script again to clear and recreate the initial database content.
-
-### Run the Backend
-
-To start the backend, run:
+Restart the api and you are good to go:
 ```
-nodemon server.js
+docker restart website-api-1
 ```
-for a development server. Navigate to `http://localhost:3002/`. The app will automatically reload if you change any of the source files.
-
->*Note that you can also run the server using `node server.js`, but it will not automatically reload if you change any of the source files.*
+Navigate to `http://localhost:3002/`. The app will automatically reload if you change any of the source files.
 
+### Applying Database Updates
+If updates or additions are made to the database design after you create the database tables for the first time, you will most likely have to apply these updates to your local databases. To do this, simply run the force_sync_db.js script again to clear and recreate the initial database content.
 ## Swagger API Reference
 
 Navigate to `http://localhost:3002/api-docs` to view the Swagger server API reference.
 
 ## Testing
 The project utilizes [Mocha](https://mochajs.org/) and [Chai](https://www.chaijs.com/) to run backend integration tests.
 
-### Set the NODE_ENV environment variable
-You will need to set the environment variable:
+Tests will start automatically when you run docker compose up -d from above. To re-run the tests, restart the testing container:
 ```
-NODE_ENV=test
+docker restart website-integration-tests-1
```

Comment:
`docker-compose up -f docker-compose.tests.yml -d --force-recreate integration-tests`

Reply:
Currently the test db and dev db run on the same container and the integration-tests service starts in the dev env as its in the same compose file.
If we move tests to a separate docker file, do you think we should also move the test db to this docker file where it can run in its own container?

Reply:
Using another compose file for test env